### ISSUE TITLE
Update wgpu-native and wgpu-py

### ISSUE THREAD

**Issue by almarklein** [pygfx/pygfx#221]

**Comment by Korijn** [comment]
Can I help with anything? Maybe:

>  Add support for aarch Linux in wgpu-native and wgpu-py (and test on a rpi).

**Comment by almarklein** [comment]
Yeah, so that part would consists of:
* Adding a build for linux-aarch64 on wgpu-native. Can do that on a fork first, and test it, before submitting an actual PR.
* Updating wgpu-py so it knows about that arch, can download it, and builds a wheel with it.

I could use a hand in the second part for sure :)

**Comment by Korijn** [comment]
Looks like this is going pretty smoothly so far?

**Comment by almarklein** [comment]
Yes, and no. Kvark had already made the necessary updates to wgpu-native. But the arm build was not really a succes.
